crates/kubelet/src/grpc_sock/windows/mod.rs

```diff
@@ -0,0 +1,91 @@
+// This is a modified version of: https://github.com/hyperium/tonic/blob/f1275b611e38ec5fe992b2f10552bf95e8448b17/examples/src/uds/server.rs
```

Comment:
Do we need to include a LICENSE or explain what changes we made to the upstream implementation?

Are the changes made here able to be patched upstream?

Basically I'm trying to determine if we need to take ownership over this codebase or if it's just a temporary fork.

Reply:
This is our own code and not a fork. Specifically, this is copied from their examples directory and tweaked to work with Windows. In these kinds of cases, a link back to the code feels fine